<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaIncluirCliente</class>
 <widget class="QDialog" name="TelaIncluirCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro De Cliente</string>
  </property>
  <widget class="QLabel" name="Fundo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Fundos/Fundos/01.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nome">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Nome :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CPF">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CPF">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>CPF :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Email :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Email">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Endereco">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Endereço</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logradouro">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Logradouro :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Numero">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string> Nº :</string>
   </property>
  </widget>
  <widget class="QLabel" name="CEP">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>CEP :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Logradouro">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>230</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Numero">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CEP">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Telefone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Telefone</string>
   </property>
  </widget>
  <widget class="QLabel" name="DDD">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>DDD :</string>
   </property>
  </widget>
  <widget class="QLabel" name="DDI">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>DDI :</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumeroFone">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Número :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DDD">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DDI">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NumeroFone">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CNH">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Carteira De Habilitação</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumeroCNH">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Número Da CNH : </string>
   </property>
  </widget>
  <widget class="QLabel" name="Categoria">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Categoria :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Validade">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(17, 17, 12);</string>
   </property>
   <property name="text">
    <string>Validade :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Categoria">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NumeroCNH">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Incluir Cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Dia">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Mes">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Ano">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BotaoVoltarMenuCliente">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
